pkg/build/api/types.go

```diff
@@ -266,6 +266,18 @@ const (
 	// secret for pushing the output image.
 	// The build will stay in the pending state until the secret is created, or the build times out.
 	StatusReasonMissingPushSecret = "MissingPushSecret"
+
+	// StatusReasonFailedPostCommitHook indicates that the postcommit failed
+	StatusReasonPostCommitHookFailed = "PostCommitHookFailed"
+
+	// StatusReasonGitCloneFailed
+	StatusReasonGitCloneFailed = "GitCloneFailed"
+
+	// StatusReasonPullImageFailed
+	StatusReasonPullImageFailed = "PullImageFailed"
```

Comment:
> ah, it is in the Status field of oc get builds:

yep, I think back then we decided not to add a new column, for it would be empty in the "ideal" scenario, or for whatever other reason.